index.html

```diff
@@ -998,16 +998,90 @@
 
 </div>
 
-## Safety Systems {#safety}
+## Safety, Abuse, and Fraud {#safety}
+
+There are some cases where privacy and safety are in tension. These cases are more rare
+than one might think at first glance, but they still exist. We can distinguish three
```

Comment:
"more rare than one might think" strikes me as speculative and indefinite. For people who work on safety, privacy presents continuous significant obstacles and destabilizes our efforts.

Reply:
Agreed, this is highly variable on how much a given reader thinks about it. I'd venture to guess that for the average privacy-minded reader, these trade-offs and abuses are not front of mind (warranting their late inclusion via this PR :])